### PR TITLE
MyBike landing

### PR DESCRIPTION
**Pull request by Taras2552**
[DEMO LINK](https://Taras2552.github.io/layout_landing-page/)

### CODE REVIEW

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]
1. add favicon
2. disable scroll when menu is open

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]
1. add favicon

**Comment by Taras2552** [comment]
> 1. add favicon
Знімок екрана 2024-02-10 о 14 52 15 (2)

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]
image
Favicon is not shown. Try to use svg format for it

File: src/index.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
     <link
     rel="icon"
     type="image/x-icon"
```
Comment (SanyaBratashchuk) [inline src/index.html:9]: change type. It may help
type="image/svg+xml"

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]

**Pull request closed without merging**